requirements-agent-release.txt

```diff
@@ -27,6 +28,7 @@ datadog-coredns==1.8.0; sys_platform == 'linux2'
 datadog-couch==4.0.0
 datadog-couchbase==1.17.0
 datadog-crio==1.6.0
+datadog-datadog-cluster-agent==1.1.0
```

Comment:
If this is new, shouldn't it be 1.0.0 instead of 1.1.0?

Reply:
it was bumped during release week with a bug fix. This file updates to latest, it wasn't added when it was initially released